apps/server/prisma/seed.ts

```diff
@@ -214,7 +181,7 @@ async function main() {
     create: {
       id: IT_EXIT_FORM_UUID,
       name: 'IT Exit Form',
-      formDocLink: IT_EXIT_FORM_LINK,
+      formDocLink: 'http://localhost:3002/test.pdf',
     },
```

Comment:
constant?

Reply:
I thought about it right before putting up the PR but I was too tired 😵‍💫. I'll add it when I get back lol.